src/satosa/micro_services/account_linking.py

```diff
@@ -53,8 +53,9 @@ def _handle_al_response(self, context):
         status_code, message = self._get_uuid(context, internal_response.auth_info.issuer, internal_response.attributes['issuer_user_id'])
 
         if status_code == 200:
-            satosa_logging(logger, logging.INFO, "issuer/id pair is linked in AL service",
-                           context.state)
+            msg = "issuer/id pair is linked in AL service"
+            logline = lu.LOG_FMT.format(id=lu.get_session_id(context.state), messge=msg)
```

Comment:
and a typo here: `message` instead of `messge`

Reply:
Thanks again